src/Exception/InvalidIfExpressionException.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sofascore\PurgatoryBundle\Exception;
+
+use Symfony\Component\ExpressionLanguage\Expression;
+use Symfony\Component\ExpressionLanguage\SyntaxError;
+
+final class InvalidIfExpressionException extends InvalidArgumentException
+{
+    private const MESSAGE = 'Invalid "if" expression provided: "%s"';
+
+    public function __construct(
+        public readonly Expression $expression,
+        SyntaxError $syntaxError,
+    ) {
```

Comment:
@Brajk19 What about adding the route name like in #106?

Reply:
@HypeMC i like it
https://github.com/sofascore/purgatory-bundle/pull/101/commits/cc7c254adf8b5fc3f7b60a054e0a9c47850e002e